.env.production

```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_API_BASE_URL=http://localhost:8000
```

Comment:
We should handcraft the `BASE_URL` kinda like how i did it here: https://github.com/Star-Sync/ss-core/blob/3584a63425e2bbc75a79557466838cb44eed3c89/sample.env#L10

but use `SS_WEB_PORT` and `SS_WEB_HOST` (and pls open a PR on `ss-deploy` to add to the sample.env